MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/labs/InsetMapActivity.java

```diff
@@ -45,7 +45,7 @@ protected void onCreate(Bundle savedInstanceState) {
       @Override
       public void onMapReady(MapboxMap mapboxMap) {
         InsetMapActivity.this.mainLargeMapboxMap = mapboxMap;
-        mainLargeMapboxMap.setOnCameraMoveListener(new MapboxMap.OnCameraMoveListener() {
+        mainLargeMapboxMap.addOnCameraMoveListener(new MapboxMap.OnCameraMoveListener() {
```

Comment:
if you add a listener, that listener should also be removed with map destruction to avoid leaking the listeners.

Reply:
Fixed